web/models/package_download.ex

```diff
@@ -2,11 +2,11 @@ defmodule HexWeb.PackageDownload do
   use HexWeb.Web, :model
   import HexWeb.QueryAPI, only: [coalesce: 2]
 
-  @primary_key false
+  @primary_key {:view, :string, []}
 
   schema "package_downloads" do
     belongs_to :package, Package, references: :id
-    field :view, :string
+    # field :view, :string
```

Comment:
I may be missing something but I think this is wrong, actually I don't know how it works :). We should add a fake primary key to the postgres view that we use as source for this schema and the use that as primary key in the schema.